README.md

```diff
@@ -32,45 +34,42 @@ See the [architecture document](docs/arch.md) for an overview of the Kusk Gatewa
 
 [(Back to top)](#table-of-contents)
 
-See our [Installation document](https://kubeshop.github.io/kusk-gateway/installation/) for how to install Kusk Gateway with Helm or how to get kusk gateway running locally.
+If you want to quickly setup and evaluate the Kusk Gateway, then please use the [local installation instructions with Minikube](local-installation.md).
 
-## Usage
+Otherwise see our [Installation document](https://kubeshop.github.io/kusk-gateway/installation/) for how to install the Kusk Gateway to Kubernetes.
 
-[(Back to top)](#table-of-contents)
+For the quick and impatient:
 
-Kusk Gateway configures itself via the API CRD that contains your embedded Swagger or OpenAPI document.
+```sh
+# Jetstack Cert-Manager MUST BE INSTALLED TO THE CLUSTER BEFORE!!
```

Comment:
This could leave users high and dry if they dont know how to install it. Could you link to external docs for how to install [here](https://cert-manager.io/v0.14-docs/installation/kubernetes/#installing-with-helm) for instance. or write a small set of steps yourself?